src/mods/depositAll/helper.js

```diff
@@ -0,0 +1,24 @@
+function deposit() {
+	// Select normal deposit button
+	document
+		.querySelector('.slot .grey.gold:not(.js-deposit-all)')
```

Comment:
Can we make all these selectors more specific?
For example, `const $stash = getWindow(WindowNames.Stash)` at the top of the methods, then all the query selectors can be `$stash.querySelector`.

Targeting such generic-sounding selectors (especially `input.formatted, `.btn.blue`) worries me that we may eventually accidentally target the wrong element.

Same feedback throughout this PR.

Reply:
Good point, fixed 'em all